.github/workflows/tag.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Are you sure this regexp works? `.` should be escaped

Reply:
absolutely